interface/forms/ros/FormROS.class.php

```diff
@@ -25,158 +42,158 @@ class FormROS extends ORDataObject {
-    var $sinus_problems = "N/A";
-    var $post_nasal_drip = "N/A";
-    var $nosebleed = "N/A";
-    var $snoring = "N/A";
-    var $apnea = "N/A";
-    var $breast_mass = "N/A";
-	var $breast_discharge = "N/A";
-	var $biopsy = "N/A";
-	var $abnormal_mammogram = "N/A";
-	var $cough = "N/A";
-	var $sputum = "N/A";
-	var $shortness_of_breath = "N/A";
-	var $wheezing = "N/A";
-	var $hemoptsyis = "N/A";
-	var $asthma = "N/A";
-	var $copd = "N/A";
-	var $chest_pain = "N/A";
-    var $palpitation = "N/A";
-    var $syncope = "N/A";
-    var $pnd = "N/A";
-    var $doe = "N/A";
-    var $orthopnea = "N/A";
-    var $peripheal = "N/A";
-    var $edema = "N/A";
-    var $legpain_cramping = "N/A";
-    var $history_murmur = "N/A";
-    var $arryhmia = "N/A";
-    var $heart_problem = "N/A";
-    var $dysphagia = "N/A";
-	var $heartburn = "N/A";
-	var $bloating = "N/A";
-	var $belching = "N/A";
-	var $flatulence = "N/A";
-	var $nausea = "N/A";
-	var $vomiting = "N/A";
-	var $hematemesis = "N/A";
-	var $gastro_pain = "N/A";
-	var $food_intolerance = "N/A";
-	var $hepatitis = "N/A";
-	var $jaundice = "N/A";
-	var $hematochezia = "N/A";
-	var $changed_bowel = "N/A";
-	var $diarrhea = "N/A";
-	var $constipation = "N/A";
-	var $polyuria = "N/A";
-	var $polydypsia = "N/A";
-	var $dysuria = "N/A";
-	var $hematuria = "N/A";
-	var $frequency = "N/A";
-	var $urgency = "N/A";
-	var $incontinence = "N/A";
-	var $renal_stones = "N/A";
-	var $utis = "N/A";
-	var $hesitancy = "N/A";
-	var $dribbling = "N/A";
-	var $stream = "N/A";
-	var $nocturia = "N/A";
-	var $erections = "N/A";
-	var $ejaculations = "N/A";
-	var $g = "N/A";
-	var $p = "N/A";
-	var $ap = "N/A";
-	var $lc = "N/A";
-	var $mearche = "N/A";
-	var $menopause = "N/A";
-	var $lmp = "N/A";
-	var $f_frequency = "N/A";
-	var $f_flow = "N/A";
-	var $f_symptoms = "N/A";
-	var $abnormal_hair_growth = "N/A";
-	var $f_hirsutism = "N/A";
-	var $joint_pain = "N/A";
-	var $swelling = "N/A";
-	var $m_redness = "N/A";
-	var $m_warm = "N/A";
-	var $m_stiffness = "N/A";
-	var $muscle = "N/A";
-	var $m_aches = "N/A";
-	var $fms = "N/A";
-	var $arthritis = "N/A";
-	var $loc = "N/A";
-	var $seizures = "N/A";
-	var $stroke = "N/A";
-	var $tia = "N/A";
-	var $n_numbness = "N/A";
-	var $n_weakness = "N/A";
-	var $paralysis = "N/A";
-	var $intellectual_decline = "N/A";
-	var $memory_problems = "N/A";
-	var $dementia = "N/A";
-	var $n_headache = "N/A";
-	var $s_cancer = "N/A";
-	var $psoriasis = "N/A";
-	var $s_acne = "N/A";
-	var $s_other = "N/A";
-	var $s_disease = "N/A";
-	var $p_diagnosis = "N/A";
-	var $p_medication = "N/A";
-	var $depression = "N/A";
-	var $anxiety = "N/A";
-	var $social_difficulties = "N/A";
-	var $thyroid_problems = "N/A";
-	var $diabetes = "N/A";
-	var $abnormal_blood = "N/A";
-	var $anemia = "N/A";
-	var $fh_blood_problems = "N/A";
-	var $bleeding_problems = "N/A";
-	var $allergies = "N/A";
-	var $frequent_illness = "N/A";
-	var $hiv = "N/A";
-	var $hai_status = "N/A";
+	public $id;
```

Comment:
@mdsupport ,
I was revisiting this change and I like the suggest of function call to clean up the original file. But you also mention converting to a LBF. Wouldn't make it an LBF would make controlling the "layout". LBF's don't always turn out as desired in my experience unless the form is done like the history or portions of the demographics that are stored in the LBF table.
The the LBF be controlled with CSS like using the js?